### ISSUE TITLE
Esquema// Estructura de una noticia para el portal Bloque64

### ISSUE THREAD

**Issue by alexaivytorres** [bloque64/bloque64_web#8]
Trabajar en el diseño, estructura y requerimientos para armonizar las noticias en el portal Bloque64

**Comment by DiazRock** [comment]
Hace falta subir al repo en JPG o PNG las componentes que se ven en el formato que @pgarciagon ha compartido. Hablo de la barra de búsqueda que se ve en la parte superior del layout, encima de la foto de BLOQUE64.

**Comment by DiazRock** [comment]
Si @pgarciagon puede decirme acerca del formato que debe tener un artículo de opinión. Ya sé que el boletín debe ser :  introducción, desarrollo, y final, con una sola foto.

**Comment by DiazRock** [comment]
Considero que para esta primera fase, si los demás tipos de noticias ya están, añadir el tipo de noticia "entrevista". Una sugerencia.

**Comment by DiazRock** [comment]
@pgarciagon @alexaivytorres , respecto a la noticia corta, cuando puedan me dejan por acá qué tipo de formato puede tener cada parte. Si la entradilla debe ser en **negrita**, o si no puede tener subrayados, y esas cosas, para cada parte de la noticia. Y si la foto (opcional) debe ir en un lugar fijo o puede cambiar de posición. Todo lo que puedan decir de la estructura de la noticia corta.

**Comment by DiazRock** [comment]
![image](https://user-images.githubusercontent.com/50834033/79900126-c25a5d80-83db-11ea-9883-2b31d8844e09.png)

**Comment by DiazRock** [comment]
Algo por ahí...

**Comment by alexaivytorres** [comment]
> Considero que para esta primera fase, si los demás tipos de noticias ya están, añadir el tipo de noticia "entrevista". Una sugerencia.

copiado

**Comment by alexaivytorres** [comment]
> Algo por ahí...

Alejandro, estoy revisando el esquema. En efecto el Titulo debe ser llamativo, pero si le das la libertad de escoger el tipo de letra será una ensalada de artículos, por lo que debemos escoger una fuente fresca, legible y seria - las más usadas arial, times- eso lo puede recomendar el diseñador.

La entradilla, podría tener un limite de palabras, cambio de color o resaltada en negritas. También podríamos seleccionar el tipo de letras. La imagen en un solo sitio. Sin embargo debemos considerar que una noticia puede tener varias imágenes. Incluso estudiar el número máximo a usar.

En el cuerpo de la noticia, una letra tal vez más pequeña, sin resaltar. Si el autor necesita resaltar una frase o palabra pudiese tener una opción.

El cierre, puede tener el mismo estilo que el contenido. 

otro aspecto a considerar es el espaciado e interlineado.

**Comment by alexaivytorres** [comment]
> > Considero que para esta primera fase, si los demás tipos de noticias ya están, añadir el tipo de noticia "entrevista". Una sugerencia.
> 
> copiado

esa entrevista podría tener dos posibilidades, escrita con soporte fotográfico o en video con breve relato o entradilla.

**Comment by alexaivytorres** [comment]
> @pgarciagon @alexaivytorres , respecto a la noticia corta, cuando puedan me dejan por acá qué tipo de formato puede tener cada parte. Si la entradilla debe ser en **negrita**, o si no puede tener subrayados, y esas cosas, para cada parte de la noticia. Y si la foto (opcional) debe ir en un lugar fijo o puede cambiar de posición. Todo lo que puedan decir de la estructura de la noticia corta.

Para una noticia corta que invite al lector sugiero una entradilla en negritas, sin subrayar, con un número de 150 palabras cuando mucho -voy a probar- la fotografía en un lugar fijo, para dar un sólo formato.

**Comment by alexaivytorres** [comment]
![promoción](https://user-images.githubusercontent.com/63795808/79906193-61d01e00-83e5-11ea-962a-17aa9ffca4db.png)

![promoción](https://user-images.githubusercontent.com/63795808/79906389-b1aee500-83e5-11ea-877f-b3256a7e3107.png)

**Comment by alexaivytorres** [comment]
> @pgarciagon @alexaivytorres , respecto a la noticia corta, cuando puedan me dejan por acá qué tipo de formato puede tener cada parte. Si la entradilla debe ser en **negrita**, o si no puede tener subrayados, y esas cosas, para cada parte de la noticia. Y si la foto (opcional) debe ir en un lugar fijo o puede cambiar de posición. Todo lo que puedan decir de la estructura de la noticia corta.

otra cosa, hay noticis cortas??? o te refieres a los titulares??? @DiazRock

**Comment by alexaivytorres** [comment]
![promoción](https://user-images.githubusercontent.com/63795808/79906704-4ca7bf00-83e6-11ea-9342-b6bb51418861.png)

**Comment by DiazRock** [comment]
> > @pgarciagon @alexaivytorres , respecto a la noticia corta, cuando puedan me dejan por acá qué tipo de formato puede tener cada parte. Si la entradilla debe ser en **negrita**, o si no puede tener subrayados, y esas cosas, para cada parte de la noticia. Y si la foto (opcional) debe ir en un lugar fijo o puede cambiar de posición. Todo lo que puedan decir de la estructura de la noticia corta.
> 
> otra cosa, hay noticis cortas??? o te refieres a los titulares??? @DiazRock 

Sí, noticias cortas. O noticias de boletín, o no sé cómo llamarles. Es el primer tipo de noticia para el que estoy creando el editor. La idea es que se escriban las cosas en un formato fijo que uno escoja con antelación:  noticia corta, artículo de opinión, etc.
